src/artemis/devices/DCM.py

```diff
@@ -0,0 +1,24 @@
+from ophyd import Component as Cpt
+from ophyd import Device, EpicsMotor, EpicsSignalRO
+
+
+class DCM(Device):
+
+    # upstream_x: EpicsMotor = Cpt(EpicsMotor, '-MO-DET-01:UPSTREAMX')
+
+    bragg: EpicsMotor = Cpt(EpicsMotor, "-MO-DCM-01:BRAGG")
+    roll: EpicsMotor = Cpt(EpicsMotor, "-MO-DCM-01:ROLL")
+    offset: EpicsMotor = Cpt(EpicsMotor, "-MO-DCM-01:OFFSET")
+    perp: EpicsMotor = Cpt(EpicsMotor, "-MO-DCM-01:PERP")
+    energy: EpicsMotor = Cpt(EpicsMotor, "-MO-DCM-01:ENERGY")
+    pitch: EpicsMotor = Cpt(EpicsMotor, "-MO-DCM-01:PITCH")
+    wavelength: EpicsMotor = Cpt(EpicsMotor, "-MO-DCM-01:WAVELENGTH")
```

Comment:
Nit: Do you know of any documentation for what each of these do? If not can we write some comments? The ones in particular I'm not so sure about:
* offset
* perp